modules/base/shaders/screenspace_fs.glsl

```diff
@@ -44,5 +45,6 @@ Fragment getFragment() {
   }
 
   frag.depth = vs_depth;
+  frag.color.rgb = pow(frag.color.rgb, vec3(1.0/(gamma + 0.000001)));
```

Comment:
I think it might be better to put that limitation onto the gamma FloatProperty,  instead of having a minimum value of 0, you can make it have a minimum value of 0.000001 instead